code-challenges.test.js

```diff
@@ -1,67 +1,249 @@
 
-// Reminder: The test will call your function
-// Run the file with the following command: $ yarn jest
+// // Use test driven development to complete the following questions
+// // Add appropriate dependencies: $ yarn add jest
 
-// --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
+// // Reminder: The test will call your function
+// // Run the file with the following command: $ yarn jest
 
-// a) Create a test with expect statements for each of the variables provided.
+// ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-const fibLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
+// // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
 
-const fibLength2 = 10
-// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+// // a) Create a test with expect statements for each of the variables provided.
 
-// b) Create the function that makes the test pass.
+// // Fibonacci Sequence: Is a series of numbers in which each number ( Fibonacci number ) is the sum of the two preceding numbers. The simplest is the series 1, 1, 2, 3, 5, 8,  the length of the array will be the length of your input
 
-// --------------------2) Create a function that takes in an object and returns an array of the values sorted from least to greatest.
-// Hint: Check out this resource: Object.values() https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values
+describe("defibrillator", () => {
+  const fibLength1 = 6
+  // Expected output: [1, 1, 2, 3, 5, 8]
+  const fibLength2 = 10
+  // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+  it("return and array whos length is equal to the value provided as an argument", () => {
+    expect(defibrillator(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+    expect(defibrillator(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+  })
+})
 
-// a) Create a test with expect statements for each of the variables provided.
+// /*
+
+//  FAIL  ./code-challenges.test.js
+//   defibrillator
+//     ✕ return and array whos length is equal to the value provided as an argument
+
+//   ● defibrillator › return and array whos length is equal to the value provided as an argument
+
+//     ReferenceError: defibrillator is not defined
+
+// */
+
+// ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+// // b) Create the function that makes the test pass.
+
+// /*
+
+// Pseudocode: 1.) Create a function called defibrillator
+// 2.) defibrillator will take an argument of a number data type and return an array
+// 3.) Create an array called fibonacciArray that has 2 elements [1, 1]
+// 4.) Iterate using a for loop 
```

Comment:
Awesome, I love the approach, and the function looks clean! I would consider adding where I will start iterating with the for loop in the pseudocode.